dependencies.yaml

```diff
@@ -86,6 +91,14 @@ dependencies:
       lang: java
       version: "1.2.17"
 
+  org.apache.commons:
+    commons-compress:
```

Comment:
anyway I can convince you to drop few dependencies added in this change :-) Looks like you really want random string utils, but rest of them may have good replacement in std lib.

Reply:
done!